db/migrations/20200203102132_create_bid_event_table.sql

```diff
@@ -1,7 +1,7 @@
 -- +goose Up
 CREATE TABLE maker.bid_event
 (
-    log_id           BIGINT PRIMARY KEY REFERENCES event_logs (id) ON DELETE CASCADE,
+    log_id           BIGINT PRIMARY KEY REFERENCES event_logs (id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED,
```

Comment:
This line fixes the flaky test we've been seeing, by specifying that the `CASCADE DELETE` should happen at the end of the transaction. This way we won't run into the issue where a trigger is trying to update a row that was already deleted, because the row won't have been deleted yet.

[Here](https://www.postgresql.org/docs/9.1/sql-set-constraints.html) are some docs about deferrable constraints if you're interested

Reply:
Don't think we want to modify the earlier migration here. Is it possible to update this column in a new migration?

Reply:
Haha oh yeah 🤦‍♂ 

Can def put this into a separate migration